operatorcourier/validate.py

```diff
@@ -371,9 +371,9 @@ def is_email(field):
             return True
 
         def is_version(field):
-            pattern1 = re.compile(r'v(\d+\.)(\d+\.)(\d)')
-            pattern2 = re.compile(r'(\d+\.)(\d+\.)(\d)')
-            return pattern1.match(field) or pattern2.match(field)
+            version = re.compile(r'v?\d+\.\d+\.\d+'
+                r'(-[A-Za-z0-9]+([-\.]?[A-Za-z0-9]+)*)?'
```

Comment:
@alecmerdler Is there a standard for what a valid CSV version field is?

Reply:
Just standard SemVer (`major.minor.patch`).

Reply:
I've seen versions such as `0.0.1` and `1.22.0-beta1`, so I've set this regex to match those as well.

Reply:
But `major.minor.patch` isn't the full standard for Semantic Versioning:

https://semver.org/#spec-item-9

Is prerelease tagging supported? If so, we should check for it.

Reply:
@kevinrizza Based on what I've seen, prerelease tagging is supported, and should be checked fo. I suggest that we use an external semver validator such as py package [semver](https://pypi.org/project/semver/)

Reply:
+1